assets/javascripts/running.js

```diff
@@ -340,7 +341,7 @@ var developerMode = {
     // state of the test execution (comes from os-autoinst cmd srv through the openQA ws proxy)
     currentModule: undefined,               // name of the current module, eg. "installation-welcome"
     moduleToPauseAt: undefined,             // name of the module to pause at, eg. "installation-welcome"
-    pauseAtTimeout: undefined,              // whether to pause on assert_screen timeout
+    pauseOnScreenMismatch: undefined,       // 'assert_screen' (to pause on assert_screen timeout) or 'check_screen' (to pause on assert/check_screen timeout)
```

Comment:
I would write it without referring to any specific function names from os-autoinst. Also in the current writing you are missing e.g. "send_key_until_needlematch" and such. However I am not sure if you actually cover "any screen mismatch" already

Reply:
It should be obvious that `check_screen`/`assert_screen` covers all functions internally calling these as well. So `check_screen` covers everything. Note that the quoted names are the values the variables is expected to be set to. I think it makes sense to call them like the os-autoinst function names and also use these in the explanation.